### PR TITLE
About page style issues

### LINKED ISSUE #127: Bug(s): `About` Page 🐛
Tasks:
- [ ] Fix the `About` page styling.

### PR DESCRIPTION
**Pull request by JamesBug404**
The about page format was fixed and some extra content (Key Features and How can it help the user) and the button to the Home page.
image

### CODE REVIEW

**Review by gracebilliris — CHANGES_REQUESTED** [review]
Can the button at the bottom of the page use the same matching colour as the other pages? Can we use Australian spelling?

**Comment by JamesBug404** [comment]
> Can the button at the bottom of the page use the same matching colour as the other pages? Can we use Australian spelling?

Yep, all fixed now.

**Review by srujankarthik — APPROVED** [review]
nice way to let the users know regarding our app with apt descriptions in the about page, good work!

File: client/src/pages/About.tsx [hunk 1]
```diff
@@ -6,22 +6,55 @@ function About(this: any) {
...
+          </p>
+        </div>
+
+        
+        <div className="mt-10 text-center" >
+          <button style={{
+                backgroundColor: "#2e7faa",
+               
+              }} 
+              className="mt-8 py-2 px-4 text-white rounded">
```
Comment (stephenb-coding) [inline client/src/pages/About.tsx]: Try not to use `style` props if you've already got `className` here

File: client/src/pages/About.tsx [hunk 2]
```diff
@@ -6,22 +6,55 @@ function About(this: any) {
...
-        navigating the digital landscape, Text Insights has you covered. We
-        prioritise simplicity, customisation, and accuracy, making complex tasks
-        a breeze. Powered by open-source models, our tool offers an ethical and
-        transparent approach to text processing. With Text Insights, you unlock
-        a world of concise understanding and sentiment analysis, enriching your
-        text-related endeavours.
-      </p>
+      <div className="mx-auto max-w-2xl p-4 mt-10 text-center"> 
+        <h1 className="text-3xl font-extrabold">About</h1> 
+        <p className="mt-4 text-sm font-bold">
```
Comment (gracebilliris) [inline client/src/pages/About.tsx]: Can we add the `id="AboutDesc"` back, please? The Cypress test is failing because it has been removed.

**Review by gracebilliris — DISMISSED** [review]

File: client/src/pages/About.tsx (hunk 1 above)
Comment (JamesBug404) [reply client/src/pages/About.tsx]: Sure, Code changed and style removed : )

File: client/src/pages/About.tsx (hunk 2 above)
Comment (JamesBug404) [reply client/src/pages/About.tsx]: I just did. Thanks Grace.

**Review by stephenb-coding — APPROVED** [review]

**Review by gracebilliris — APPROVED** [review]
Looks good!

**Pull request merged**